pages/docs/reference/coding-conventions.md

```diff
@@ -746,7 +746,7 @@ class Person(
 Using trailing commas has several benefits:
 
 * It makes version-control diffs cleaner – as all the focus is on the changed value.
-* It makes adding and reordering elements – there is no need to add or delete the comma if you manipulate elements.
+* It makes it easier to add and reorder elements – there is no need to add or delete the comma if you manipulate elements.
 * It simplifies code generation, for example, for object initializers. The last element can also have a comma.
 
 Trailing commas are entirely optional – your code will still work without them. The Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it optional for the call site.
```

Comment:
Can we avoid using the word "optional" twice here with two slightly different meanings?

Reply:
```suggestion
Trailing commas are entirely optional – your code will still work without them. The Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it at your discretion for the call site.
```